saml/services/default-profile-mapper.js

```diff
@@ -0,0 +1,60 @@
+import { uid, email, givenname as firstName, surname as lastName, } from "./saml-constants/claims.js"
+import { unspecified } from "./saml-constants/name-identifer-formats.js"
+
+const FieldClaimMapping = {
+    uid,
```

Comment:
if using userID as key then we can only use default mapper instead of having special one for bioland?
```suggestion
    userID: uid,
```